exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -111,6 +112,10 @@ private Adapter() {}
             .setVType(Model.ValueType.INT64)
             .build());
 
+    if (!span.getStatus().isOk() && !span.getAttributes().containsKey(KEY_ERROR)) {
```

Comment:
```suggestion
    if (!span.getStatus().isOk()) {
```
I think it may be more important that the jaeger `error` tag always maps to span status, than preserve an attribute named `error` that the user may set on the span.

Unfortunately, we don't have a spec to resolve this kind of thing yet (see https://github.com/open-telemetry/opentelemetry-specification/issues/414), but that issue does reference the python implementation, and I think this is how they've implemented it (+/- my python reading skill).

Reply:
You are correct @trask, we don't want to add new semantic conventions for Jaeger only.

Reply:
The logic there should be based on the span status as per the suggested change. The additional check was just because it is possible to add duplicate `Model.KeyValue` entries as it is a `List`, unlike the attributes `Map` that it is built from. I'll accept your suggested changes as the Otel Collector and Jaeger appear to accept this behaviour without failure. Thanks for that.

Reply:
Btw, thanks @sleighzy for all of the auto-instrumentation related testing you've been doing!